traffic_monitor/health/event.go

```diff
@@ -42,8 +42,8 @@ type Event struct {
 	Hostname      string `json:"hostname"`
 	Type          string `json:"type"`
 	Available     bool   `json:"isAvailable"`
-	IPv4Available bool   `json:"isAvailable"`
-	IPv6Available bool   `json:"isAvailable"`
+	IPv4Available bool   `json:"isIPv4Available"`
+	IPv6Available bool   `json:"isIPv6Available"`
```

Comment:
This should be `ipv6Available` to match the JSON for `tc.IsAvailable.Ipv6Available`:
https://github.com/apache/trafficcontrol/blob/730d842d07d736c344adad9f5081ec30e6790ea9/lib/go-tc/crstates.go#L42